README.md

```diff
@@ -0,0 +1,37 @@
+# Taxi Service
+This project implements a taxi service, with which you can register
+and subsequently authenticate as a driver,
+after which various functions will be available, such as:  </br>
+* Display All Drivers
+* Display All Cars
+* Create new Car
+* Create new Manufacturer
+* Add Driver to Car
+* etc.
```

Comment:
let's remove etc and add all available functions, also, let's combine similar paragraphs like display in one paragraph